### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by emahuni**
fix: typos and gramma

### CODE REVIEW

**Comment by emahuni** [comment]
There are a few things that are note clear here...

This example: 
```vue
<div v-transfer-dom:header.replace>This <div> will replace the node with the `header` id, wherever it is in the DOM</div>
```
is not using the target param, but yet you said:

> Please note that this modifier will only be valid when used with `target` param.

Your implementation of the `replace` modifier behaviour is ok, however, when I suggested this I was of the impression that it would clear the contents of the target then become its only child instead of destroying the target. I think we can call this new feature i just mentioned here `clear` so that it is different in a sense and leave replace doing what you just made it do.
It could do it this way; in fact I had implemented virtually all of this functionality. Just having a few bugs here and there (in the remove function)
so it could just do this:
```js
let child;
while((child = parentNode.firstChild)) {
child.remove()
}
```
 before it becomes the only child. 


### classes and multiple elements
You need to make clones of source node should there be multiple nodes that result from the queryselect.

**Comment by tinymins** [comment]
Yeah, seems that it is a mistake of saying `modifier will only be valid when used with target param`, since `target` field can be set from not only `params` but also from modifier.

`<div v-transfer-dom:header.replace>` equals with `<div v-transfer-dom.replace="{ target: '#header' }">`

**Comment by emahuni** [comment]
ok, so we can remove that limitation notice then?

**Comment by tinymins** [comment]
In fact, that may be dangerous. Because if you do not set target, the default target will be `body`, which means it will remove whole `body` with this transfer-dom.

File: README.md [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
 ```
 
 ## usage
 
-You can simply add a directive tag to start using.
+Simply add a directive tag on the element that you wish to transfer.
 
 ```html
-<div v-transfer-dom>This text will be transfered to end of body</div>
+<div v-transfer-dom>This <div> will be transfered to the end of <body>'s children</div>
```
Comment (tinymins) [inline README.md:27]: Shouldn't `<div>` and `<body>` be escaped?

Comment (tinymins) [reply README.md:27]: or just remove `<` and `>`?

Comment (emahuni) [reply README.md:27]: yah i guess you could escape or remove them

Comment (tinymins) [reply README.md:27]: > yah i guess you could escape or remove them

But do I have any way to modify your pull request commit code? I haven't do this before 😅

Comment (emahuni) [reply README.md:27]: yes most certainly;

Screenshot 2019-07-12 at 19 21 31

 you see that, it means that you can make edits.

Comment (emahuni) [reply README.md:27]: just edit the file.

Screenshot 2019-07-12 at 19 22 54

**Pull request merged**